.env.example

```
@@ -1,8 +1,14 @@
+# This file will be committed to version control, so make sure not to have any secrets in it.
+# If you are cloning this repo, create a copy of this file named `.env` and populate it with your secrets.
+
 # Environment
 NODE_ENV=development
 
 ## OPENAI
 OPENAI_API_KEY=""
 
 ## Metadata
-VERCEL_PROJECT_PRODUCTION_URL=""
+NEXT_PUBLIC_BASE_URL=""
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)